Gemfile

```diff
@@ -1,3 +1,2 @@
 source 'https://rubygems.org'
 gemspec
-gem 'rspec'
```

Comment:
Removed this since it was causing this deprecation:

> Your Gemfile lists the gem rspec (>= 0) more than once.
> You should probably keep only one of them.
> Remove any duplicate entries and specify the gem only once (per group).
> While it's not a problem now, it could cause errors if you change the version of one of them later.